Furball.Game/Screens/BasicTestScreen.cs

```diff
@@ -28,6 +31,37 @@ public class BasicTestScreen : Screen {
 
             this.Manager.Add(whiteTexture);
 
+            UiButtonDrawable testButton = new("test button", ContentReader.LoadRawAsset("default-font.ttf"), 50, Color.White, Color.Black) {
+                Position      = new Vector2(200, 200),
+                ColorOverride = Color.Blue
+            };
+            testButton.OnHover += delegate {
```

Comment:
the formatting here is a bit ass ngl, id do this:

```
testButton.Tweens.Add(
    new VectorTween(
        TweenType.Movement,
        testButton.Position,
        new(testButton.Position.X + FurballGame.Random.NextSingle(-50, 50), testButton.Position.Y + FurballGame.Random.NextSingle(-50, 50)),
        FurballGame.Time,
        FurballGame.Time + 1000
     )
);
```

Reply:
resolved in 9b9dfb360f668114899d1f7310cb0526ea8a8c17